scripts/push-translations.bash

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -e
+
+printf "Pushing Translations
+-----------------------------
+"
+tx push -s
```

Comment:
This is kinda useless?

Reply:
Sure, it just serves as a kind of bookmark, as in I don't kneed to know what program and/or options to invoke.

Reply:
@imsodin :
tx push -s uploads the english strings (for all resources, the so called default locale) to  transifex so other's can start translating new or changed strings. Maybe exit code should be checked if the cmd succeeded.

Reply:
I know what it does, my point is that if I want to get strings to transifex I don't necessarily remember `tx push -s` but looking at tab completion I would consider `push-translations.bash` as a tool that likely does what I want to :)

Reply:
Ahh now I understand :-). Yes I agree this easier to remember and also better for coding newbies because they won't need to dig into transifex docs first to know what is what (sometime ago I felt lost at this point I remember and looking at the scripts helped a lot to learn.)